docs/quickstart.md

```diff
@@ -35,7 +35,7 @@ Everytime you want to add a new repository to gitbase, the application should be
 Run the [latest released version of the frontend](https://hub.docker.com/r/srcd/gitbase-playground/tags/):
 
 ```bash
-$ GITBASEPG_REPOS_FOLDER=./repos docker-compose up --force-recreate
+$ GITBASEPG_REPOS_FOLDER=./repos make compose-serve-latest
```

Comment:
Please, let's keep quickstart depend only on single docker/docker-compose tool commands - having some magic make files however nice, is not transparent.

Reply:
I'm not sure if it's a good idea to expose that complexity, but rebased, squashed, and added your suggestion.